pyproject.toml

```diff
@@ -81,8 +81,8 @@ weaviate-client = {version = ">=3.22", optional = true}
 adlfs = {version = ">=2022.4.0", optional = true}
 pyodbc = {version = "^4.0.39", optional = true}
 qdrant-client = {version = "^1.6.4", optional = true, extras = ["fastembed"]}
-databricks-sql-connector = {version = ">=2.9.3,<3.0.0", optional = true}
-dbt-databricks = {version = ">=1.7.3", optional = true}
+databricks-sql-connector = "3.1.2"
```

Comment:
Any rason to pin this specific versions? Otherwise we should just set lower bound on the minimum version that works. E.g. `^3` or `^3.1`  
Also make sure to keep the `optional` flag